overlays/panda/prod/cnpg-panda-server-database.yaml

```diff
@@ -0,0 +1,45 @@
+
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  # note the name is the DB services/pods  not the Cluster name.
+  name: usdf-panda-server
+spec:
+  instances: 1
+  primaryUpdateStrategy: unsupervised
+
+  bootstrap:
+    initdb:
+      database: panda
+      owner: panda
+      secret:
+        name: usdf-panda-server-creds
+  
+  imageName:  ghcr.io/cloudnative-pg/postgresql:14.4-7
```

Comment:
For PanDA database, I am not sure this image will work.
PanDA database has some special configuration https://github.com/PanDAWMS/panda-database/blob/main/etc/postgresql.conf#L3-L5.

Reply:
hm not sure how to do that / add that, is there any example that I can look at from rucio or butler or anywhere?

Reply:
Here there is an example https://github.com/slaclab/rubin-usdf-butler-deploy/blob/main/Dockerfile.

Reply:
ah wonderful, thanks!